ansible/roles/ci-admin-user/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: ci-admin-user | Create ci-admin group
+  group:
+    name: ci-admin
+    state: present
+  become: yes
+
+- name: ci-admin-user | Create ci-admin user
+  user:
+    name: ci-admin
+    comment: "CI Administrator for Ansible use"
+    group: ci-admin
+    password: "{{ ci_admin_password_hash }}"
+    state: present
+  become: yes
+
+- include: freebsd.yml
+  when: ansible_os_family == 'FreeBSD'
```

Comment:
IIRC last week you wanted to get rid of the "skipped tasks" messages. Does something like `include: "{{ ansible_os_family }}.yml"` work (and create possibly empty `{{ ansible_os_family }}.yml` files for the other supported OSes)?